cdap-ui/app/directives/macro-widget-toggle/macro-widget-toggle.html

```diff
@@ -40,7 +40,7 @@
       -->
       <fieldset
         ng-disabled="MacroWidget.disabled"
-        ng-if="(['javascript-editor', 'python-editor', 'scala-editor'].indexOf(MacroWidget.field['widget-type']) === -1)"
+        ng-if="(MacroWidget.editorTypeWidgets.indexOf(MacroWidget.field['widget-type']) === -1)"
```

Comment:
I think there are a few other places in this html that can be refactored with `MacroWidget.editorTypeWidgets`